easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'PEAR'
+version = '0.9.11'
+
+homepage = 'https://cme.h-its.org/exelixis/web/software/%(namelower)s/'
+description = """PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
+ It is fully parallelized and can run with as low as just a few kilobytes of memory."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+# Needs manual download via web-form,
+# https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis
+# source_urls = ['https://cme.h-its.org/exelixis/web/software/pear/files/']
```

Comment:
Since now it can only be manually downloaded, I see little point leaving this URL here

Reply:
@vanzod I deleted the line.